common/djangoapps/util/milestones_helpers.py

```diff
@@ -313,6 +314,18 @@ def remove_content_references(content_id):
     return milestones_api.remove_content_references(content_id)
 
 
+def any_unfulfilled_milestones(course_id, user_id):
+    """ Returns a boolean if user has any unfulfilled milestones """
+    from milestones.exceptions import InvalidMilestoneRelationshipTypeException
```

Comment:
So, if the milestones app is not installed, and you call any_unfulfilled_milestones, the function will bomb due to the missing exceptions module -- you'll need to short-circuit this function as the others do, based on the feature flag.